crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -1494,8 +1494,8 @@ bitflags::bitflags! {
         const SCREEN_SPACE_REFLECTIONS          = 1 << 16;
         const HAS_PREVIOUS_SKIN                 = 1 << 17;
         const HAS_PREVIOUS_MORPH                = 1 << 18;
-        const OIT_ENABLED                       = 1 << 18;
-        const LAST_FLAG                         = Self::HAS_PREVIOUS_MORPH.bits();
```

Comment:
Naive question: why does this constant exist?

Reply:
Looking below, apparently it's used to calculate the offset for MSAA_SHIFT_BITS, etc.